com.unity.render-pipelines.core/Runtime/XR/XRSystem.cs

```diff
@@ -0,0 +1,374 @@
+using System;
+using System.Collections.Generic;
+
+#if ENABLE_VR && ENABLE_XR_MODULE
+using UnityEngine.XR;
+#endif
+
+namespace UnityEngine.Rendering
+{
+    /// <summary>
+    /// Used by render pipelines to communicate with XR SDK.
+    /// </summary>
+    public static class XRSystem
+    {
+        // Valid empty pass when a camera is not using XR
+        public static readonly XRPass emptyPass = new XRPass();
+
+        // Keep track of only one XR layout
+        static XRLayout s_Layout = new XRLayout();
+
+    #if ENABLE_VR && ENABLE_XR_MODULE
+        static List<XRDisplaySubsystem> s_DisplayList = new List<XRDisplaySubsystem>();
+        static XRDisplaySubsystem s_Display;
+    #endif
+
+        // MSAA level (number of samples per pixel) shared by all XR displays
+        static int s_MsaaLevel = 1;
```

Comment:
Can we use the `MSAASamples` enum for this?

Reply:
Good point, done!